code/cli/src/operational.rs

```diff
@@ -0,0 +1,624 @@
+use composable_traits::vesting::{VestingScheduleInfo, VestingWindow};
+use primitives::currency::CurrencyId;
+use sp_core::crypto::Ss58Codec;
+use sp_runtime::{codec::Compact, AccountId32};
+use ss58_registry::Ss58AddressFormatRegistry;
+use substrate_api_client::{
+	compose_call, compose_extrinsic, rpc::WsRpcClient, Api, AssetTipExtrinsicParams,
+	GenericAddress, XtStatus,
+};
+
+const INFRASTRUCTURE_PROVIDERS: &[(&str, Amount<Raw>)] = &[
+	("5xD9TpbabV6VdC2F9RL5yQA61DPzWfWn1U6b76ckZBeQL2vo", Amount::new(1_000_000)),
+	("5wyRiyzsTWvXcb7BpHYjDs7GgK3VpbccRTdonv2Wvow7GAkg", Amount::new(2_962_963)),
+	("5CGCdbYh9KwmSM8NYPCRuz4tUqnbfAUtNy4Ei5Rg2qujatqR", Amount::new(1_000_000)),
+	("5zEddnhFgz8yyAXwMDew9SMNQveGmcEP73CThfYh6DfkQXMH", Amount::new(8_000_000)),
+	("5zEddnhFgz8yyAXwMDew9SMNQveGmcEP73CThfYh6DfkQXMH", Amount::new(2_000_000)),
+	("16A5r6WjR2DxE5iezedrEEf4aSrKkNMwF1Py8ir52pk1Z5oN", Amount::new(1_481_481)),
+	("DzepVXv9Z6RGKTJUWj6jECqaRh7gpYc7sGSYdG6Yzoj2dWm", Amount::new(2_518_519)),
+	("162LiM9SQSPUBQW8ZUivmg8oTE4DHKDDrNpf7TUDg4RdVQbj", Amount::new(2_000_000)),
+	("5ECzzYfSwuSd64mYu2Vn42NnqVCm2o9HtGi6Q7NLuBqewZDd", Amount::new(2_000_000)),
+	("5uPfUKgKc76LN2t6Cj7BVAnY5trNm8YK23Cgn4kwWWoYu7AL", Amount::new(2_000_000)),
+	("5D215FWsfUqiBUYbtioxcb6mCbY5AFcUimPhhiF9xHieoTs1", Amount::new(2_000_000)),
+	("5HT2WdLNgm6vDa5Xdx6dyPAGL2Qtsy4f6HGkpnGjtC1u4Ms4", Amount::new(1_000_000)),
+	("5DLTAmm2tcm4zGX1r9wGae5LCC5L2JHqXu2resqg5e3PgALj", Amount::new(8_000_000)),
+	("5DLTAmm2tcm4zGX1r9wGae5LCC5L2JHqXu2resqg5e3PgALj", Amount::new(4_000_000)),
+	("5FHPEDBTdfyqX4MnLwSJ7F2mp9e9Uhsx8fshxSL6KBwjSTWD", Amount::new(1_000_000)),
+	("5vSR8xyCVUtD2qiHEPiG9owNnFxpUQfzEfK8AAuAYAW9aGoR", Amount::new(1_000_000)),
+	("5vSR8xyCVUtD2qiHEPiG9owNnFxpUQfzEfK8AAuAYAW9aGoR", Amount::new(4_000_000)),
+	("5yhcepLAujpJcQfLMkZdLE4tPLBjyqaSCiQq1NvYak6D9rD1", Amount::new(2_000_000)),
+	("1BkvLWCxkvrmiTjogjRgUpfDV7TCRYRkPamAyHgxpvZBrFG", Amount::new(2_000_000)),
+	("1BkvLWCxkvrmiTjogjRgUpfDV7TCRYRkPamAyHgxpvZBrFG", Amount::new(1_000_000)),
+	("5Cevx2zUE4PcqtF1wMHr1msDWvXkZuFtEfG5crEp3SYp5bNF", Amount::new(2_000_000)),
+	("5Cevx2zUE4PcqtF1wMHr1msDWvXkZuFtEfG5crEp3SYp5bNF", Amount::new(1_000_000)),
+	("5HYdhBoTQR7fdqFLeZuMzHn2GktSUyZyKrr8spQb7i94etru", Amount::new(2_888_889)),
+	("5tmsbdYgTBwQMZ1QbsN1STUV58DeNSW9nQZSHdjd46ezeArM", Amount::new(1_000_000)),
+	("5vdQNCfbwinB5jYp2EfxUPpj3Cq1nJMDWnJUXcKfYs2LYzuh", Amount::new(2_456_140)),
+	("15tHCtqr2ct42aaN4oiw2yVfiPUutGLzKMcjnwX5Y5tskwMx", Amount::new(2_105_263)),
+	("5xHPbqATyTk9hkFwjYP8duX9Qy3T1PRUtF6WJdM9GPoAyZpf", Amount::new(1_500_000)),
+	("5xHPbqATyTk9hkFwjYP8duX9Qy3T1PRUtF6WJdM9GPoAyZpf", Amount::new(1_000_000)),
+	("15nY6N7RiWtVhEAX8jNYWPgNEiFrATw93SLsufdBw2XUtAAo", Amount::new(4_108_333)),
+	("5yP5uDcTmQMPsgUPyB86j3pxmDo27mrWbiJCjd2cBrkfopAY", Amount::new(3_111_111)),
+];
+
+const INVESTORS: &[(&str, Amount<Raw>)] = &[
+	("5v5AKbVaHzCx3p56gdnXHczQehRZv6s32rUT2Es2jVfzne9N", Amount::new(25000000)),
```

Comment:
These 2 addresses are the same as 52 is KSM version for 51. When populating the vesting pallet we identify 2 vesting schedule IDs for the same wallet ( 51 and 52). Please reviese this as this came directly from Master excel sheet.

Reply:
Fixed in the master sheet

Reply:
And updated the code accordingly